src/components/AddForm.js

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import { useState } from 'react';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import CancelIcon from '@mui/icons-material/Cancel';
+import Grid from '@mui/material/Grid';
+
+const AddForm = ({onAdd, setOpen}) => {
+  const [titleNote, setTitleNote] = useState("");
+  const [contentNote, setContentNote] = useState("");
+  const handleTitleChange = (event) => {
+    setTitleNote(event.target.value);
+  };
+  const handleContentChange = (event) =>{
+    setContentNote(event.target.value);
+  };
+  const [title, setTitle] = useState('');
+  const [content, setContent] = useState('');
```

Comment:
Can you explain to me why we have titleNote and title, and contentNote and content?

Reply:
TitleNote and ContentNote are not required in the code, so I will delete them. Title and content disable the add button if the required fields are not filled